app/javascript/controllers/address_fill_controller.js

```diff
@@ -10,7 +10,7 @@ export default class extends Controller {
       name: this.data.get("name"),
       source: (query, populateResults) => {
         const results = []
-        ajax({
+        Rails.ajax({
```

Comment:
I think something changed so that `ajax` is no longer exported so needed to update these references